client/src/modules/inventory/configuration/groups/modals/actions.modal.js

```diff
@@ -89,8 +89,8 @@ function InventoryGroupsActionsModalController(InventoryGroups, Notify, Instance
 
     if (vm.identifier) {
       InventoryGroups.read(vm.identifier)
-        .then(function (groups) {
-          vm.session = groups[0];
+        .then((groups) => {
+          vm.session = groups;
```

Comment:
Are you sure this doesn't break inventory?  What changed that it should be `groups` instead of `groups[0]` now?

Reply:
Yes, there was a bug with `group[0]`